packages/vscode-extension/src/service/VSCommandOutputHandler.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+
+import { scloudOutput } from '../extension';
+import OperationResult from '../OperationResult';
+import { NodeUtils, unwrapExceptionMessage, unwrapInformationMessage } from '../util/ExtensionUtil';
+
+export default class VSCommandOutputHandler {
```

Comment:
We should re-use CommandOutputHandler from node-cli package

Reply:
CommandOutputHandler is deleted and instead we are ussing directly the loggers